spec/spec_helper_acceptance.rb

```diff
@@ -2,6 +2,7 @@
 require 'beaker-puppet'
 require 'beaker/puppet_install_helper'
 require 'beaker/module_install_helper'
+require 'camptocamp/systemd'
```

Comment:
You have to remove this line.
The module `camptocamp/systemd` have to be installed with `spec_helper_acceptance.rb` if required by acceptances tests.

Reply:
Like here : https://github.com/voxpupuli/puppet-etherpad/blob/master/spec/spec_helper_acceptance.rb#L14